tests/test_codebase/test_mmdet3d/data/smoke.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
can we use image from `demo/resources` instead of uploading `tests/test_codebase/test_mmdet3d/data/nuscenes/n015-2018-07-24-11-22-45+0800__CAM_BACK__1532402927637525.jpg`?

Reply:
Could we use the existing files instead of add `tests/test_codebase/test_mmdet3d/data/nuscenes/n015-2018-07-24-11-22-45+0800.pkl`

Reply:
hi, where do you use `tests/test_codebase/test_mmdet3d/data/nuscenes/n015-2018-07-24-11-22-45+0800__CAM_BACK__1532402927637525.jpg`?